modules/troubleshooting-dspa-component-errors.adoc

```diff
@@ -0,0 +1,221 @@
+:_module-type: REFERENCE
+
+[id="troubleshooting-dspa-component-errors_{context}"]
+= Troubleshooting DSPA component errors
+
+[role='_abstract']
+This table displays common errors found in DataSciencePipelinesApplication (DSPA) components, along with their statuses, messages, and proposed solutions. The **Ready** condition type accumulates errors from various DSPA components, providing a status view of the DSPA deployment.
+
+[cols="10%,10%,20%,60%", options="header",]
+|===
+|Type |Status |Error message |Solution
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`Could not connect to Object Store: tls: failed to verify certificate: x509: certificate signed by unknown authority`
+|This issue occurs in clusters that use self-signed certificates with {productname-short} version 2.9 or later. The data science pipelines manager cannot connect to the object storage as it does not trust the object storage SSL certificate. Therefore, the pipeline server cannot be created. 
+
+Contact your IT operations administrator to add the relevant Certificate Authority bundle. 
+
+ifdef::upstream[]
+For more information, see link:{odhdocshome}/installing-open-data-hub/#understanding-certificates_certs[Understanding certificates in {productname-short}].
+endif::[]
+
+ifdef::self-managed,cloud-service[]
```

Comment:
Will the cloud-service[] condition use the ifndef::disconnected or ifdef::disconnected?